src/parsers/sample-models.hxx

```diff
@@ -1,6 +1,7 @@
 //
 // Copyright (c) 2015-2019 CNRS INRIA
 // Copyright (c) 2015 Wandercraft, 86 rue de Paris 91400 Orsay, France.
+// Copyright (c) 2023 Heriot-Watt University
```

Comment:
Concerning this copyright, there is no IP concerning the change you made.
Please remove it.

Reply:
Done